src/bundle/Resources/views/content/locationview.html.twig

```diff
@@ -170,6 +174,7 @@
         '@EzPlatformAdminUiBundle/Resources/public/js/scripts/button.state.toggle.js'
         '@EzPlatformAdminUiBundle/Resources/public/js/scripts/admin.version.edit.conflict.js'
         '@EzPlatformAdminUiBundle/Resources/public/js/scripts/admin.location.bookmark.js'
+        '@EzPlatformAdminUiModulesBundle/Resources/public/js/ContentTree.module.js'
```

Comment:
Is it working? The JS which needs this file is added before. This should be moved to the `react_modules` block.